core/src/main/java/org/testcontainers/containers/Container.java

```diff
@@ -130,6 +130,15 @@ public String getStderr() {
     SELF withFileSystemBind(String hostPath, String containerPath, BindMode mode);
 
     /**
+     * Adds container volumes.
+     *
+     * @param otherContainer the other container object to add volumes from
+     * @param mode the bind mode
+     * @return this
+     */
+    SELF withVolumesFrom(String otherContainer, BindMode mode);
```

Comment:
IMO it should have an overloaded method with `Container` as a first parameter, the same as `withLink`